src/gridstack-poly.js

```diff
@@ -202,4 +219,46 @@ if (!Array.prototype['forEach']) {
     }
     // 8. return undefined
   };
-}
\ No newline at end of file
+}
+
+// https://developer.mozilla.org/zh-CN/docs/Web/API/CustomEvent/CustomEvent
+(function () {
+  if (window.navigator.userAgent.indexOf('Chrome') > -1 ) {
+    return false
+  }
+  
+  function CustomEvent ( event, params ) {
+      params = params || { bubbles: false, cancelable: false, detail: undefined };
+      var evt = document.createEvent('CustomEvent');
+      evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
+      return evt;
+  }
+
+  CustomEvent.prototype = window.Event.prototype;
+
+  window.CustomEvent = CustomEvent;
+})();
+
+//https://github.com/jserz/js_piece/blob/master/DOM/ChildNode/remove()/remove().md
+(function (arr) {
+  arr.forEach(function (item) {
+   if (item.hasOwnProperty('remove')) {
+     return;
+   }
+   Object.defineProperty(item, 'remove', {
+     configurable: true,
+     enumerable: true,
+     writable: true,
+     value: function remove() {
+       this.parentNode.removeChild(this);
+     }
+   });
+ });
+})([Element.prototype, CharacterData.prototype, DocumentType.prototype]);
+
+// suport IE
+(function () {
+  if (!document.scrollingElement) {

```

Comment:
interesting way to support field name change... is there a problem caching this though ? I'm ok with caller doing 
document.scrollingElement || document.documentElement
instead.